helix-loader/build.rs

```diff
@@ -2,15 +2,21 @@ use std::borrow::Cow;
 use std::path::Path;
 use std::process::Command;
 
-const MAJOR: &str = env!("CARGO_PKG_VERSION_MAJOR");
-const MINOR: &str = env!("CARGO_PKG_VERSION_MINOR");
-const PATCH: &str = env!("CARGO_PKG_VERSION_PATCH");
-
-fn get_calver() -> String {
-    if PATCH == "0" {
-        format!("{MAJOR}.{MINOR}")
+fn get_calver() -> &'static str {
+    if env!("CARGO_PKG_VERSION_PATCH") == "0" {
+        concat!(
+            env!("CARGO_PKG_VERSION_MAJOR"),
+            ".",
+            env!("CARGO_PKG_VERSION_MINOR")
+        )
     } else {
-        format!("{MAJOR}.{MINOR}.{PATCH}")
+        concat!(
+            env!("CARGO_PKG_VERSION_MAJOR"),
+            ".",
+            env!("CARGO_PKG_VERSION_MINOR"),
+            ".",
+            env!("CARGO_PKG_VERSION_PATCH")
+        )
     }
```

Comment:
this change isn't worth it in my opinion. The `format` version is more readable.

Reply:
Okay, I see